rust/osm2lanes/src/locale.rs

```diff
@@ -0,0 +1,78 @@
+use serde::{Deserialize, Serialize};
+
+/// Context about the place where an OSM way exists.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Locale {
+    pub driving_side: DrivingSide,
+    /// When sidewalks are not explicitly tagged on a way, should sidewalks or shoulder lanes be
+    /// placed anyway based on heuristics?
+    pub infer_sidewalks: bool,
+}
+
+impl Locale {
+    pub fn builder() -> Config {
+        Config::default()
+    }
+}
+
+/// Configuration to build locale, context about the place where an OSM way exists.
+#[derive(Default)]
+pub struct Config {
```

Comment:
So defaults for lane widths, bikes allowed in bus lanes, typical colors for lane markings, etc can all live here. Sounds good to me.